src/azure-cli-core/azure/cli/core/cloud.py

```diff
@@ -241,19 +241,6 @@ def get_custom_clouds(cli_ctx):
     return [c for c in get_clouds(cli_ctx) if c.name not in known_cloud_names]
 
 
-def init_known_clouds(force=False):
```

Comment:
Why is this not needed anymore?

Reply:
I removed it a while back.
https://github.com/Azure/azure-cli/commit/29983162f211ae9fd353268b6ff70161673d8fc3#diff-69bacf5348b997023cd1e8e6f475a0da

It may have been missed in a bad merge.